azkaban-web-server/src/main/java/cloudflow/docs/ExecutionDetailSample.md

```diff
@@ -0,0 +1,194 @@
+{
+  "executionId": "2",
+  "submitUser": "azkaban",
+  "submitTime": 1581383285656,
+  "concurrentOption": "skip",
+  "failureAction": "FINISH_CURRENTLY_RUNNING",
+  "experimentId": "defaultExperimentId",
+  "description": "defaultDescription",
+  "previousFlowExecutionId": "defaultPreviousExecutionId",
+  "notifyFailureFirst": false,
+  "notifyFailureLast": false,
+  "rootFlow": {
+    "name": "flowPriority2",
+    "type": "ROOT_FLOW",
+    "startTime": 1581383286307,
+    "endTime": 1581383587055,
+    "status": "SUCCEEDED",
+    "flowInfo": {
+      "flowId": "defaultId",
+      "flowName": "flowPriority2",
+      "flowVersion": "1"
+    },
+    "nodeList": [
+      {
+        "name": "jobD",
+        "type": "JOB",
+        "startTime": 1581383436938,
+        "endTime": 1581383436965,
+        "status": "SUCCEEDED"
+      },
+      {
+        "name": "jobE",
+        "type": "JOB",
+        "startTime": 1581383436979,
+        "endTime": 1581383587011,
+        "status": "SUCCEEDED"
+      },
+      {
+        "name": "emb_1",
+        "type": "EMBEDDED_FLOW",
+        "startTime": 1581383286307,
+        "endTime": 1581383286581,
+        "status": "SUCCEEDED",
+        "flowInfo": {
+          "flowId": "defaultId",
+          "flowName": "flowPriority2:emb_1",
+          "flowVersion": "1"
+        },
+        "nodeList": [
+          {
+            "name": "jobE1B",
+            "type": "JOB",
+            "startTime": 1581383286531,
+            "endTime": 1581383286560,
+            "status": "SUCCEEDED"
+          },
+          {
+            "name": "jobE1A",
+            "type": "JOB",
+            "startTime": 1581383286402,
+            "endTime": 1581383286494,
+            "status": "SUCCEEDED"
+          }
+        ]
+      },
+      {
+        "name": "emb_2",
+        "type": "EMBEDDED_FLOW",
+        "startTime": 1581383286400,
+        "endTime": 1581383436933,
+        "status": "SUCCEEDED",
+        "flowInfo": {
+          "flowId": "defaultId",
+          "flowName": "flowPriority2:emb_2",
+          "flowVersion": "1"
+        },
+        "nodeList": [
+          {
+            "name": "jobE2C",
+            "type": "JOB",
+            "startTime": 1581383436888,
+            "endTime": 1581383436922,
+            "status": "SUCCEEDED"
+          },
+          {
+            "name": "jobE2B",
+            "type": "JOB",
+            "startTime": 1581383286516,
+            "endTime": 1581383286541,
+            "status": "SUCCEEDED"
+          },
+          {
+            "name": "emb_3",
+            "type": "EMBEDDED_FLOW",
+            "startTime": 1581383286560,
+            "endTime": 1581383436881,
+            "status": "SUCCEEDED",
+            "flowInfo": {
+              "flowId": "defaultId",
+              "flowName": "flowPriority2:emb_2:emb_3",
+              "flowVersion": "1"
+            },
+            "nodeList": [
```

Comment:
Wondering if something like "executables" would be a better name for this field (I'm looking for something closer to the business domain and less close to the implementation details and the data structure being used).

Reply:
I was also thinking about an alternative that avoids the term `node` but didn't quite find anything which is less confusing than `nodeList`. I am open to changing this but not sure if `executables` is the best alternative.

Reply:
ok, let's go with nodeList for now and think about a better name.